src/pydicom/pixels/encoders/native.py

```diff
@@ -38,6 +40,11 @@ def _encode_deflated_frame(src: bytes, runner: EncodeRunner) -> bytes:
     bytes
         A deflate encoded frame.
     """
+    # In the case of single bit images, the data must first be bit-packed
+    # before being encoded with Deflate
```

Comment:
Is there a reference in the standard for this?

Reply:
Maybe [C.8.20.2](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.8.20.2.html#sect_C.8.20.2.1) in part 3?

Reply:
I'm just wondering if all bits stored/bits allocated data must be bit-packed or if its only segmentation data.